Cargo.toml

```diff
@@ -11,6 +11,9 @@ members = [
     "x/programs/test/programs/*",
     "x/programs/cmd/simulator/test/*",
 ]
+default-members = [
+    "x/programs/rust/wasmlanche-sdk",
+]
```

Comment:
From the docs: 
>Packages to operate on when a specific package wasn’t selected.

I'm not sure why we would want this, we want to operate on the entire workspace when a package isn't specified. 

```suggestion
```

Reply:
Yeah, If I run `cargo test` it doesn't run for the whole workspace. This isn't desired behaviour.